src/store/reducers/Surveys/index.test.ts

```diff
@@ -0,0 +1,34 @@
+import { SurveysState, initialState } from '.';
+import { surveysReducers } from './actions';
+
+describe('surveys slice', () => {
+  describe('nextSurvey', () => {
+    it('changes the current position to next position', () => {
+      const mockState: SurveysState = { ...initialState };
```

Comment:
Later, we should record the HTTP request to simulate this instead of using the mocking data.
You can check [`pollyJS`](https://github.com/Netflix/pollyjs) to implement it

Reply:
@hoangmirs oh, but this is not a API calling request 😅 Thanks, noted it 🙆‍♂️